varya/assets/sass/elements/_forms.scss

```diff
@@ -41,3 +41,16 @@ input[type=checkbox] + label {
 	margin-left: 0.5em;
 	line-height: 1em;
 }
+
+.search-form {
+	display: flex;
+	margin-bottom: var(--global--spacing-vertical);
```

Comment:
Let’s remove the bottom margin here. Vertical spacing is already applied by the parent element in this file which covers both the widget and elements on the 404 page inside `.page-content`: 

https://github.com/Automattic/themes-workspace/blob/master/varya/assets/sass/structure/_vertical-margins.scss